src/store/createStore.ts

```diff
@@ -39,6 +39,11 @@ type TransactionBuildParamsObj = {
   [K in keyof TransactionBuildParams]?: TransactionBuildParams[K];
 };
 
+export enum SignTxActiveView {
+  import = "import",
+  overview = "overview",
+}
+
```

Comment:
We could use a string union instead of an enum in cases like this. That would give us better hints in code editors.